schemas/event.yml

```diff
@@ -38,53 +38,174 @@
+          description: >
+            The signal value is used by Elastic Kibana apps, such as SIEM, for app-specific purposes.
+            `event.kind:signal` is thus reserved and should not be used for the ingestion
+            of events into Elasticsearch.
 
     - name: category
       level: core
       type: keyword
-      short: Event category.
+      short: Event category. (Categorization Field)
       description: >
-        Event category.
-
-        This contains high-level information about the contents of the event. It
-        is more generic than `event.action`, in the sense that typically a
-        category contains multiple actions.
-      example: user-management
-      accepted_values:
-      - name: authentication
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-        expected_event_types:
-          - allow
-          - deny
-          - info
-      - name: process
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-        expected_event_types:
-          - start
-          - info
-          - end
+        This is one of four ECS Categorization Fields, and indicates the
+        second level in the ECS category hierarchy.
+
+        `event.category` represents the "big buckets" of ECS categories. For example,
+        filtering on `event.category:process` yields all events relating to process activity.
+        This field is closely related to `event.type`, which is used as a subcategory.
+      example: authentication
+      allowed_values:
+        - name: authentication
+          description: >
+            Events in this category are related to the challenge and response process
+            in which credentials are supplied and verified to allow the creation of a session.
+            Common sources for these logs are Windows event logs, ssh logs, etc.
+            Visualize and analyze events in this category to look for unusual login activity,
+            failed logins, etc.
+          expected_event_types:
+            - start
+            - end
+            - info
+        - name: database
+          description: >
+            The database category denotes events and metrics relating to a data storage
+            and retrieval system. Note that use of this category is not limited to
+            relational database systems. Examples include event logs from MS SQL, MySQL,
+            Elasticsearch, MongoDB, etc. Use this category to visualize and analyze
+            database activity such as accesses and changes.
+          expected_event_types:
+            - access
+            - change
+            - info
+            - error
+        - name: driver
+          description: >
+            Having to do operating system device drivers and similar software entities
```

Comment:
Should we make these into full sentences too? Something like:

> Events in the driver category have to do with ...

Reply:
Yeah I think these all need more fleshing out. If we mark all of this as beta and subject to change, I think we can keep refining and backport to 1.4 aggressively, so the docs website gets the improvements even after the release.

Otherwise we typically only backport fixes like obvious typos or inconsistencies and so on.

Reply:
Agreed that full sentences will read better in the documentation.